infrastructure/kube/keep-test/geth-node/eth-goerli-statefulset.yaml

```
@@ -0,0 +1,38 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: geth-goerli
```

Let's add labels, this would be useful e.g. for log filtering in the GCP web console.
```suggestion
  name: geth-goerli
  labels:
    app: geth
    network: goerli
```